src/stores/VoiceRecordingStore.ts

```diff
@@ -44,48 +49,56 @@ export class VoiceRecordingStore extends AsyncStoreWithClient<IState> {
         return;
     }
 
+    public getVoiceRecordingId(room: Room, relation?: IEventRelation): string {
+        if (relation?.rel_type === "io.element.thread" || relation?.rel_type === RelationType.Thread) {
+            return room.roomId + SEPARATOR + relation.event_id;
+        } else {
+            return room.roomId;
+        }
+    }
+
```

Comment:
This can, and should be, a static method.

```suggestion
    public static getVoiceRecordingId(room: Room, relation?: IEventRelation): string {
        if (relation?.rel_type === "io.element.thread" || relation?.rel_type === RelationType.Thread) {
            return room.roomId + SEPARATOR + relation.event_id;
        } else {
            return room.roomId;
        }
    }

```

You’d then call it directly with `VoiceRecordingStore.getVoiceRecordingId(.,.)`